senders/msteams/msteams.go

```diff
@@ -0,0 +1,208 @@
+package msteams
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/moira-alert/moira"
+	"github.com/russross/blackfriday/v2"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+// Sender implements moira sender interface via MS Teams
+type Sender struct {
+	frontURI string
+	logger   moira.Logger
+	location *time.Location
+	client   *http.Client
+}
+
+// Init initialises settings required for full functionality
+func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+	sender.logger = logger
+	sender.location = location
+	sender.frontURI = senderSettings["front_uri"]
+	sender.client = &http.Client{
+		Timeout:   time.Duration(30) * time.Second,
+		Transport: &http.Transport{DisableKeepAlives: false},
```

Comment:
Hi @imavroukakis! Thank you for the changes you made here! Let's remove this redundant transport declaration to not distort anyone who will read this code later.

This declaration is redundant because by default `Transport` will be `nil` here and `http.Client` will use default transport as written here https://golang.org/src/net/http/client.go#L193.

The default transport is decalred here https://golang.org/src/net/http/transport.go?#L42 and `DisableKeepAlives` is not set to any value which leads us to `DisableKeepAlives` be set to false.

Reply:
@litleleprikon 👍

Reply:
@litleleprikon done

Reply:
@imavroukakis Perfect! Thank You!